packages/runner/lib/server.ts

```diff
@@ -51,3 +43,18 @@ function runProcedure() {
             return await exec(nangoProps, input.isInvokedImmediately, input.isWebhook, code, codeParams);
         });
 }
+
+export const server = express();
+server.use(timeout('24h'));
+server.use(
+    '/',
+    trpcExpress.createExpressMiddleware({
+        router: appRouter,
+        createContext: () => ({})
+    })
+);
+server.use(haltOnTimedout);
+
+function haltOnTimedout(req: Request, _res: Response, next: NextFunction) {
+    if (!req.timedout) next();
+}
```

Comment:
switching the server to use express instead of node.http so we can have server side timeout via middleware